### DISCUSSION TITLE
Network Graph functionality question

### DISCUSSION THREAD

**Discussion by wennerma** [airbnb/visx#1158] [Q&A]
Is it possible to add nodes without supplying coordinates? I want to display a dynamic number of nodes and the logic involved to calculate that on the fly seems quite difficult.

**Comment by williaster** [comment]
Hey @wennerma 👋  , currently no. You may be interested in this [related discussion comment](https://github.com/airbnb/visx/discussions/1130#discussioncomment-602696) which gives more context.

**Reply by wennerma** [reply]
Thanks! I’ll take a look.
